dev/inquirer_common.sh

```diff
@@ -111,9 +116,23 @@ gen_index() {
   fi
 }
 
-control_c() {
+cleanup() {
+  # Reset character attributes, make cursor visible, and restore
+  # previous screen contents (if possible).
+  tput sgr0
   tput cnorm
+  tput rmcup || clear
```

Comment:
While this is a good idea for some usage cases, it might not result in intended behavior. For example, in ./examples/showcase_example.sh, I intended the behavior of the terminal such that the screen is not restored / cleared halfway through user input (making it somewhat like filling in a form). I think perhaps it would be best for the user to flank the input library calls with `tput smcup` / `tput rmcup` if they so wish to, and not put it in the cleanup function.

Reply:
Sounds good.